### PR TITLE
CSS Practice Homework

### PR DESCRIPTION
**Pull request by kmundle**

### CODE REVIEW

**Comment by tkraak** [comment]
Well done so far! All your _month / year_ list items are now sitting on their own line. First, let's understand why that is and then let's fix it. Hint: what's the _inline_ equivalent of the `<div>` tag? 😉

**Comment by kmundle** [comment]
I saw that... I was just hoping it was supposed to be that way, haha. It looked nice!

File: kelly-mundle.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  <head>
+    <title>CSS Timeline</title>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <meta name="description" content="A Timeline of CSS"> <!--added for clarity-->
+    <link rel="stylesheet" href="/kelly-mundle.css">
+  </head>
+  <body>
+    <h1>A Brief History of CSS</h1> <!--some minor edits... ha-->
+      <ul>
+        <li><div id="list-item-1"><span class="date">1994:</span> The first work on style sheets started at CERN.</div></li>
```
Comment (tkraak) [inline kelly-mundle.html]: What's this `<div>` for?

Comment (kmundle) [reply kelly-mundle.html]: To turn the first list item green with an ID selector- if you're referencing line 11.

Comment (tkraak) [reply kelly-mundle.html]: Yes, line 11 ... the `<div>` is not needed at all ... you are overthinking it 😎

Comment (kmundle) [reply kelly-mundle.html]: I believe I remedied it. Used span again, instead. I was taking slides too literally!

**Comment by tkraak** [comment]
How about this?

```html
<li id="list-item-1"><span class="date">1994:</span> The first work on style sheets started at CERN.</li>
```

**Comment by kmundle** [comment]
ooooooh. and this is preferred as it is always good to be more specific if possible, yes? Is there an instance span would be a problem?

**Comment by tkraak** [comment]
You simply don't need the extra element. That's all there is to it 😃 .

**Comment by kmundle** [comment]
I didn't realize that using that would remove the standalone li! ok ok, this makes sense.

File: kelly-mundle.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+              <ul>
+                <li>Stylesheet for printing on paper may need to be different from one for presenting information on a screen.</li>
+                <li>CSS2 lets authors associate different stylesheets for different media with the same document.</li>
+                <li>Authors can control page breaks.</li>
+                <li>Has widow and orphan control, just as for ordinary desktop publishing.</li>
+                <li> For high-quality printing, crop marks can be printed outside the outer edges of the page box.</li>
+              </ul>
+            <li>Better specification of fonts for a document.</li>
+              <ul>
+                <section>
```
Comment (tkraak) [inline kelly-mundle.html]: According to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul), `<section>` is not allowed inside `<ul>`.

**Pull request merged**